pkg/asset/installconfig/gcp/validation.go

```diff
@@ -96,6 +97,12 @@ func ValidateInstanceType(client API, fieldPath *field.Path, project, region str
 		allErrs = append(allErrs, field.Invalid(fieldPath.Child("type"), instanceType, errMsg))
 	}
 
+	typeArch := mapiutil.CPUArchitecture(instanceType)
```

Comment:
Is it worth it to make the call after the `len(arch) > 0` is done or does it make no difference?

Reply:
In this particular case it is not much of a difference but that's a good call anyway. I'll change it if we decide to go with the import-mapi approach.